src/PhpSpec/Formatter/CurrentExampleWriter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Formatter;
+
+use PhpSpec\Console\IO;
+use PhpSpec\Message\CurrentExample;
+
+class CurrentExampleWriter
+{
+    private $output;
+
+    public function __construct(IO $io)
+    {
+        $this->output = $io;
+    }
+
+    public function displayFatal(CurrentExample $currentExample, $error = null)
+    {
+        if (!empty($error) && $currentExample->getCurrentExample())
+        {
+            $this->output->writeln(sprintf('<failed>✘ %s</failed>', "Fatal error happened while executing the following"));
```

Comment:
This needs a non-decorated callback for plaintext consoles (see how most Formatters handle this). You can check isDecorated on the IO.